src/react/components/forms/CreateDealForm.tsx

```diff
@@ -27,11 +29,17 @@ export const CreateDealForm = (props: Props) => {
 		if (wallet?.publicKey && connection.connection) {
 			setPlaceholder("0");
 			setBorrower(wallet?.publicKey.toString());
+			updateLiquidityPoolBalance(); 
 		} else {
 			setPlaceholder("Connect wallet");
 		}
 	}, [connection.connection, wallet?.publicKey]);
 
+	const updateLiquidityPoolBalance = async () => {
```

Comment:
doesn't it complain we're not using `useCallback` here?

Reply:
no, but feel free to change if it's common practice

Reply:
no, if it doesn't complain then it's prob not necessary